README.md

```diff
@@ -30,6 +30,33 @@ There are plenty of small, self-contained examples [in the documentation][exampl
 
 The easiest way to get started using ct is to download one of the binaries from the [releases page on GitHub][releases].
 
+One can use the following script to download and verify the signature of Config Transpiler:
+
+```bash
+# Sepcify Config Transpiler version
+CT_VER=v0.6.1
+# Sepcify OS
+OS=apple-darwin # MacOS
+OS=unknown-linux-gnu # Linux
+# Specify download URL
+DOWNLOAD_URL=https://github.com/coreos/container-linux-config-transpiler/releases/download
+
+# Remove previous downloads
+rm -f /tmp/ct-${CT_VER}-x86_64-${OS} /tmp/ct-${CT_VER}-x86_64-${OS}.asc /tmp/coreos-app-signing-pubkey.gpg
+
+# Download Config Transpiler binary
+curl -L ${DOWNLOAD_URL}/${CT_VER}/ct-${CT_VER}-x86_64-${OS} -o /tmp/ct-${CT_VER}-x86_64-${OS}
```

Comment:
It would be nice if the arch was also a variable, so users could set `ARCH=aarch64` or something like that